KsApi/models/lenses/UserLenses.swift

```diff
@@ -15,6 +15,7 @@ extension User {
         location: $1.location,
         name: $1.name,
         needsFreshFacebookToken: $1.needsFreshFacebookToken,
+        needsPassword: $1.needsPassword,
```

Comment:
good call updating these, didn't think to do that, but they are important like we say with the project currency pr.